openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml

```diff
@@ -151,9 +151,16 @@
   implementation_status: complete
   narrative:
     - text: |-
-        A control response is planned. Engineering progress can be tracked via:
+        OpenShift offers support for replay-resistant authentication
```

Comment:
@carlosmmatos @JAORMX could you please double or triple check this? The way I read the control, as long as TLS is used, we're good, in other words, the control doesn't care about the actual mechanism underneath (it doesn't consider the attack vector where someone replays the plaintext because it doesn't consider the attacker having access to the plaintext)

Reply:
I agree with your assessment, I think we meet this. Are we gonna do an OpenSCAP check or do we claim to inherently meet this since TLS is enabled from day 1?

Reply:
We have https://issues.redhat.com/browse/CMP-940 to check TLS for IDPs. All other IDP types require TLS, htpasswd is not allowed by other controls. If you can think about other checks we can do, I'm all for doing them, but I couldn't think of any more myself.

Reply:
sounds good to me. I'll merge this.